test/screenshot/infra/commands/build.js

```diff
@@ -59,6 +59,7 @@ class BuildCommand {
       return;
     }
 
+    await this.setRunningProcessId_();
```

Comment:
When is this ever cleared? If I'm running `npm start`, the file gets created, then when I kill it, the file never gets deleted. Wouldn't that cause a false positive?

Reply:
The file never gets deleted, but I avoid false positives by checking if the process is actually still running on startup.

I could conceivably delete the file within a process `exit` handler, with the caveat that it obviously won't get called if the node process abends. WDYT?

Reply:
I missed the ps check before. Is it possible for us to verify that the running process is actually what we think it is and not coincidentally another process that spawned with the same ID?

Reply:
I tried to do that by making sure the running process is `node` with an argument of `build`:

https://github.com/material-components/material-components-web/blob/0b46c6b7a057e12fb1caab08c718fd281afb01c1/test/screenshot/infra/commands/build.js#L166-L168

https://github.com/material-components/material-components-web/blob/0b46c6b7a057e12fb1caab08c718fd281afb01c1/test/screenshot/infra/lib/process-manager.js#L108-L113

Reply:
Oh that's what commandName and argumentPattern end up being. OK.

Reply:
I added a bit of code to delete the PID file when the process exits.